src/panels/CharmsAndActionsPanel/CharmActionsPanelTitle/CharmActionsPanelTitle.test.tsx

```diff
@@ -86,7 +86,6 @@ describe("CharmActionsPanelTitle", () => {
 
     const title = screen.getByText("1 application (2 units) selected");
     expect(title).toBeVisible();
-    expect(title.tagName).toBe("H5");
```

Comment:
This element doesn't exist in this component as it was moved to the panel header.